protos/feast/core/Registry.proto

```diff
@@ -37,6 +38,7 @@ message Registry {
     repeated OnDemandFeatureView on_demand_feature_views = 8;
     repeated RequestFeatureView request_feature_views = 9;
     repeated FeatureService feature_services = 7;
+    repeated SavedDataset saved_datasets = 11;
```

Comment:
Do you want comments in the PR or in the RFC? Is there anything preventing SavedDataset from just being `Dataset`? Or is the idea that this could eventually be generalized?

Reply:
I prefer comments here. I don't think it could be generalized, at least I don't see use cases right now. I think logged dataset should be a separate instance.